quilt/test/test_install.py

```diff
@@ -125,3 +125,15 @@ def _mock_package(self, package, pkg_hash, contents, hashes):
     def _mock_s3(self, pkg_hash, contents):
         s3_url = 'https://example.com/%s' % pkg_hash
         self.requests_mock.add(responses.GET, s3_url, contents)
+
+# execute outside of mock scope so we get real network
+def test_prod_install():
+    # public package
+    command.install('akarve/days', force=True)
+    from quilt.data.akarve import days
+    df = days.names.data()
+    assert(df.loc[3]['Day'] == 'Wednesday')
+    # TODO clean up after this test b/c tests shouldn't have side-effects
```

Comment:
Create a temporary directory, `chdir`, run `install`, remove the directory - same as what `QuiltTestCase` does.